hs_communities/views/communities.py

```diff
@@ -104,9 +105,18 @@ class TopicsView(TemplateView):
     """
 
     def get(self, request, *args, **kwargs):
+        u = User.objects.get(pk=self.request.user.id)
+        if u.username not in ['czo_national', 'czo_sierra', 'czo_boulder', 'czo_christina', 'czo_luquillo', 'czo_eel',
```

Comment:
Could you throw in a TODO for making this editable in the admin page?

Reply:
https://github.com/hydroshare/hydroshare/issues/3622